patches/extra/ungoogled-chromium/keep-expired-flags.patch

```diff
@@ -0,0 +1,29 @@
+--- a/components/flags_ui/flags_state.cc
++++ b/components/flags_ui/flags_state.cc
+@@ -577,14 +577,15 @@ void FlagsState::GetFlagFeatureEntries(
+   int current_platform = GetCurrentPlatform();
+ 
+   for (const FeatureEntry& entry : feature_entries_) {
++    std::string desc = entry.visible_description;
+     if (skip_feature_entry.Run(entry))
```

Comment:
Based on my reading of `chrome/browser/about_flags.cc`, specifically `ShouldSkipConditionalFeatureEntry()`, I believe that `skip_feature_entry.Run(entry)` may return true when the entry has not expired, but rather is intended to be skipped for another reason. For example, upstream intends that the `enable-unsafe-webgpu` flag is only available on the dev and canary release channels; this, along with other conditions, is enforced in `ShouldSkipConditionalFeatureEntry()`.

Reply:
Good point, I'll update the patch to check explicitly for expired flags along with the verbiage changes.  Interestingly the `enable-unsafe-gpu` flag is available without these changes.  I was worried that there may be an issue with some other ungoogled-chromium patch causing an issue, but I checked with a build of vanilla chromium and the flag is available there too!

Reply:
Our builds likely count as `Channel::UNKNOWN`, which would explain why `enable-unsafe-gpu` is available.